lib/common/templates/ios-swift/enum.swift.template

```diff
@@ -0,0 +1,39 @@
+<%
+//
+// Copyright 2019 Alaska Air Group or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+%>
+//
+// <%= this.destination %>
+//
+<%
+    // for backward compatibility we need to have the user explicitly hide it
+    var showFileHeader = (this.options && this.options.hasOwnProperty('showFileHeader')) ? this.options.showFileHeader : true;
+    if(showFileHeader) {
+        print("// Do not edit directly\n");
+        print("// Generated on " + new Date().toUTCString());
+    }
+%>
+//
+<%
+  // Filter to only those props wanted based on the filter, then sort
+  // them by category so we keep like props together, then by name
+  // so they are easier to find alphabetically.
+  var filteredProps = _.filter(allProperties, this.filter); 
```

Comment:
I'm 99% sure you can remove this, I believe when we properly added the filter functionality we actually run the filters before the format so the format code itself doesn't have to worry about implementing filters. [Take a look here](https://github.com/amzn/style-dictionary/blob/master/lib/buildFile.js#L45)